### PR TITLE
feat(gnb-profile): remove currency selectDropdown

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
The Currency SelectDropdown is removed.

image

### CODE REVIEW

File: apps/web/src/common/modules/navigations/gnb/modules/gnb-profile/GNBProfile.vue [hunk 1]
```diff
@@ -214,33 +189,33 @@ const handleClickSignOut = async () => {
...
+                <!--                                </div>-->
+                <!--                            </div>-->
+                <!--                        </div>-->
+                <!--                    </div>-->
+                <!--                    <p-i :name="state.currencyMenuVisible ? 'ic_chevron-up' : 'ic_chevron-down'"-->
+                <!--                         class="arrow-icon"-->
+                <!--                         width="1rem"-->
+                <!--                         height="1rem"-->
+                <!--                    />-->
+                <!--                </div>-->
```
Comment (piggggggggy) [inline apps/web/src/common/modules/navigations/gnb/modules/gnb-profile/GNBProfile.vue:218]: question: r u going to remove this later?

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Review by yuda110 — APPROVED** [review]
LGTM...! But there's an error in review action..

File: apps/web/src/common/modules/navigations/gnb/modules/gnb-profile/GNBProfile.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/common/modules/navigations/gnb/modules/gnb-profile/GNBProfile.vue:218]: Nope, This code might need recovery in version 1.13.
I didn't leave any comments... I'll add some!

**Review by WANZARGEN — APPROVED** [review]
LGTM!

**Pull request merged**